From 58fa47459f995717d24e823783387453b71326ec Mon Sep 17 00:00:00 2001
From: Khalid Faisal Ansari <khalid.ansari@oss.qualcomm.com>
Date: Fri, 14 Aug 2026 17:09:18 +0530
Subject: [PATCH] arm64: dts: qcom: talos-lyra-evk: Enable TPM (ST33)

Enable dTPM (ST33HTPH2X32AHE4) over SPI6 on the Talos Lyra EVK
by adding the required SPI and TPM nodes.

Signed-off-by: Khalid Faisal Ansari <khalid.ansari@oss.qualcomm.com>
Upstream-Status: Pending
---
 arch/arm64/boot/dts/qcom/talos-lyra-evk.dts | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/talos-lyra-evk.dts b/arch/arm64/boot/dts/qcom/talos-lyra-evk.dts
index 8820886b2101..38be5d12e74e 100644
--- a/arch/arm64/boot/dts/qcom/talos-lyra-evk.dts
+++ b/arch/arm64/boot/dts/qcom/talos-lyra-evk.dts
@@ -310,3 +310,12 @@ &usb_2 {
 
 	status = "okay";
 };
+&spi6 {
+        status = "okay";
+
+        tpm@0 {
+                compatible = "st,st33htpm-spi", "tcg,tpm_tis-spi";
+                reg = <0>;
+                spi-max-frequency = <20000000>;
+        };
+};
-- 
2.34.1

